style.css

```diff
@@ -0,0 +1,68 @@
+/* CSS for Section 2 */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #000000;
+}
+
+.section2 {
+  font-family: Poppins, -apple-system, system-ui, "Segoe UI", Oxygen, Ubuntu,
+    Cantarell, "Open Sans", "Helvetica Neue,";
+  letter-spacing: normal;
+  margin: 0 250px;
+}
+
+.imglogo {
+  align-content: center;
+  margin: 80px 270px 0 310px;
+  width: 256px;
+  height: 75px;
+  fill: none;
+}
+
+div h1 {
+  text-align: center;
+  line-height: 90px;
+  font-size: 70px;
+  display: block;
+  -webkit-text-fill-color: transparent;
+  -webkit-background-clip: text;
+  background-image: linear-gradient(
+    292.46deg,
+    #27ae60 17.79%,
+    #2f80ed 35.92%,
+    #9b51e0 54.06%,
+    #eb5757 71.08%,
+    #f2994a 87.36%
+  );
+}
+
+div h2 {
+  color: #eee;
+  text-align: center;
+  line-height: 30px;
+  font-size: 26px; 
+  float: none;
+  position: static;
+  z-index: auto;
+  margin: 15px;
+}
+
+.js-sample {
+  height: 489px;
+  width: 752px;
+  margin: auto;
+}
+
```

Comment:
instead of calling the p call the class you gave it coz other P's are used in the entire work